turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -1374,28 +1377,41 @@ struct FindPackageResult {
 #[turbo_tasks::function]
 async fn find_package(
     lookup_path: FileSystemPath,
-    package_name: RcStr,
+    package_name: Pattern,
     options: Vc<ResolveModulesOptions>,
 ) -> Result<Vc<FindPackageResult>> {
     let mut packages = vec![];
     let mut affecting_sources = vec![];
     let options = options.await?;
+    let package_name_cell = Pattern::new(package_name.clone());
```

Comment:
Do we want to change `read_matches` to take a `Pattern` instead of a `Vc<Pattern>`?